providers/providerBase.php

```diff
@@ -126,4 +126,9 @@ public function getAPIInfo() {
             'VERSION' => $this->APIVersion,
         );
     }
+
+    public function LogInfo($msg)
+    {
+        freepbx_log(FPBX_LOG_INFO, $msg);
+    }
```

Comment:
I understand this change but would like to retain freepbx_log everywhere that it was used. Why? because freepbx_log is used throughout the FreePBX code and is clearly understood. By defining this helper function it becomes less clear when reading the code and we have to look up what LogInfo means.